frontend/public/index.html

```diff
@@ -3,6 +3,12 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link
+  rel="stylesheet"
+  href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+  integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+  crossorigin="anonymous"
```

Comment:
Vlad, this bootstrap https://react-bootstrap.github.io/

Reply:
Unde l-ai pus ca nu vad nimic instalat in `package.json` ? Trebuie sa te duci cu CLI in folderul cu frontend si sa dai `yarn install react-bootstrap bootstrap`, si dupa pui dependentele pt stil ...